docs_sphinx/user/nmlimport.rst

```diff
@@ -0,0 +1,322 @@
+
+                <member segment="4"/>
+            </segmentGroup>
+
+            <segmentGroup id="apical1" neuroLexId="sao864921383">
+
+                <member segment="5"/>
+            </segmentGroup>
+
+            <segmentGroup id="basal0" neuroLexId="sao864921383">
+
+                <member segment="6"/>
+            </segmentGroup>
+
+            <segmentGroup id="basal1" neuroLexId="sao864921383">
+
+                <member segment="7"/>
+            </segmentGroup>
+
+            <segmentGroup id="basal2" neuroLexId="sao864921383">
+
+                <member segment="8"/>
+            </segmentGroup>
+
+            <segmentGroup id="all">
+                <include segmentGroup="soma"/>
+                <include segmentGroup="apical0"/>
+                <include segmentGroup="apical2"/>
+                <include segmentGroup="apical3"/>
+                <include segmentGroup="apical4"/>
+                <include segmentGroup="apical1"/>
+                <include segmentGroup="basal0"/>
+                <include segmentGroup="basal1"/>
+                <include segmentGroup="basal2"/>
+            </segmentGroup>
+
+            <segmentGroup id="soma_group" neuroLexId="GO:0043025">
+
+                <include segmentGroup="soma"/>
+            </segmentGroup>
+
+            <segmentGroup id="dendrite_group" neuroLexId="GO:0030425">
+
+                <include segmentGroup="apical0"/>
+                <include segmentGroup="apical2"/>
+                <include segmentGroup="apical3"/>
+                <include segmentGroup="apical4"/>
+                <include segmentGroup="apical1"/>
+                <include segmentGroup="basal0"/>
+                <include segmentGroup="basal1"/>
+                <include segmentGroup="basal2"/>
+            </segmentGroup>
+
+            <segmentGroup id="apical_dends">
+                <include segmentGroup="apical0"/>
+                <include segmentGroup="apical2"/>
+                <include segmentGroup="apical3"/>
+                <include segmentGroup="apical4"/>
+                <include segmentGroup="apical1"/>
+            </segmentGroup>
+
+            <segmentGroup id="middle_apical_dendrite">
+                <include segmentGroup="apical3"/>
+            </segmentGroup>
+
+            <segmentGroup id="thalamic_input">
+                <include segmentGroup="apical1"/>
+            </segmentGroup>
+
+            <segmentGroup id="basal_dends">
+                <include segmentGroup="basal0"/>
+                <include segmentGroup="basal1"/>
+                <include segmentGroup="basal2"/>
+            </segmentGroup>
+
+            <segmentGroup id="basal_gaba_input">
+                <include segmentGroup="basal0"/>
+            </segmentGroup>
+
+            <segmentGroup id="background_input">
+                <include segmentGroup="basal1"/>
+            </segmentGroup>
+        </morphology>
+    </cell>
+
+Handling FractionAlong value
```

Comment:
I would rather explain this in the section title (e.g. "Handling sections not connected at the distal end")